### ISSUE TITLE
Missing module?

### ISSUE THREAD

**Issue by Umeaboy** [samuellb/fribid#1]
[kristoffer@localhost fribid (master)]$ ./configure

```
Prefix:            /usr/local
Plugin path:       /usr/lib64/mozilla/plugins

PKCS#11 smartcard support enabled
PKCS#11 module:    /usr/lib64/opensc-pkcs11.so
```

ERROR: PKCS#11 module not found

Where might I find this module?

**Comment by Umeaboy** [comment]
I found the solution to this problem by installing opensc & lib64opensc-devel. Therefor now closing this issue as FIXED. If anyone feels that this is wrong, please feel free to open this issue again & add new info.

**Comment by samuellb** [comment]
It's part of OpenSC... If you don't need smartcard support you can disable it with --disable-pkcs11

Regards,
Samuel

**Comment by Umeaboy** [comment]
I don't mean to be rude, but shouldn't that be told as you compile it?

**Comment by samuellb** [comment]
Yes I agree :)  I've improved the error message so it asks you to install OpenSC

**Comment by samuellb** [comment]
I've fixed a bug in the PKCS#11 detection that I introduced with the last change, sorry!

Please pull if you encounter problems
